content/blog/example-of-mapreduce-with-ezmeral-data-fabric-mapr-database-binary-table.md

```diff
@@ -0,0 +1,291 @@
+                BYTES_IN_REMOTE_RESULTS=0
+                BYTES_IN_RESULTS=0
+                MILLIS_BETWEEN_NEXTS=0
+                NOT_SERVING_REGION_EXCEPTION=0
+                NUM_SCANNER_RESTARTS=0
+                NUM_SCAN_RESULTS_STALE=0
+                REGIONS_SCANNED=0
+                REMOTE_RPC_CALLS=0
+                REMOTE_RPC_RETRIES=0
+                ROWS_FILTERED=0
+                ROWS_SCANNED=0
+                RPC_CALLS=0
+                RPC_RETRIES=0
+        Shuffle Errors
+                IO_ERROR=0
+        File Input Format Counters
+                Bytes Read=0
+        File Output Format Counters
+                Bytes Written=0
+```
+
+**Note**: If you encountered an issue indicating that some of the HBase related Java packages are missing, you can simply copy the following packages from 📁<ins>/opt/mapr/hbase/hbase-{VERSION}/lib/</ins> to 📁<ins>/opt/mapr/hadoop/hadoop-{VERSION}/share/hadoop/common/</ins>.
+
+* hbase-client-*.jar
+* hbase-server-*.jar
+* hbase-protocol-*.jar
+* hbase-hadoop2-compat-*.jar
+* hbase-hadoop-compat-*.jar
+* hbase-metrics-*.jar
+* hbase-metrics-api-*.jar
+* hbase-shaded-gson-*.jar
+* hbase-shaded-htrace-*.jar
+* metrics-core-*.jar
+* hbase-common-*.jar
+
+You may run into this issue because the HBase related packages are considered as third-party libraries to the Hadoop system.
+For more information, refer to👉: [Install the third-party libraries on each node that runs the program](https://docs.datafabric.hpe.com/70/DevelopmentGuide/Manage3rdPartyLibsForMapReduce.html).
+
+## Glossary
+
+<details>
+<summary>HPE Ezmeral Data Fabric (aka. MapR)</summary>
+
+HPE Ezmeral Data Fabric is a platform for data-driven analytics, ML, and AI workloads.
+The platform serves as a secure data store and provides file storage, NoSQL databases, object storage, and event streams.
+The patented filesystem architecture was designed and built for performance, reliability, and scalability.
+📖[Documentation website](https://docs.datafabric.hpe.com/70/index.html)
+
+</details>
+
+<details>
+<summary>Ezmeral Ecosystem Packs</summary>
+
+This is a software collection package that includes computing scheduling frameworks and computing engines of common Hadoop ecosystems such as YARN, Spark, Drill, and Hive, as well as a service suite for monitoring performance indicators and logs of Ezmeral Data Fabric.
+Users of Ezmeral Data Fabric will use these customized Spark, Drill, Hive and other software to complete computing, analysis tasks and machine learning tasks.
+
+</details>
+
+## Summary
+
+In this blog post, I introduced the advantages of Ezmeral Data Fabric Database compared to similar products. I also demonstrated a MapReduce application based on Ezmeral Data Fabric Database binary table.
+HPE Ezmeral Data Fabric Database binary table is equivalent to a better alternative to Apache HBase.
+
+I hope this blog post can help you quickly develop applications based on Ezmeral Data Fabric Database binary table.
```

Comment:
Don't forget the "HPE" in front of Ezmeral Data Fabric and "datbase" as opposed to "Datebase"